Agenda for 2024-07-17 CG meeting (#674)
solid/specification: meetings/2024-07-17.md

For reference, please have a look at the TPAC 2023 videos: https://www.w3.org/2023/09/TPAC/group-updates.html

* HZ: Does anybody plan to be present or prepare a demo? 

> * eP: Is it part of the breakout session.

Reviewer: ```suggestion
* eP: Is it part of the breakout session?
```

* HZ: I think the videos can be seen outside of the session. 

> * eP: I have some videos for SAI so I can prepare some short videos, will look at the guidelines. 

Reviewer: ```suggestion
* eP: I have some videos for SAI, so I can prepare some short videos; will look at the guidelines. 
```

* HZ: Please let me know if I can help.
* eP: I can do it as an individual but open to do it as shared CG effort. 

> * RG: What is the plan of CG or Solid team. What will be the structure of this session. I can update on work on notifications. If the plan is to talk about administration it probably will not fit.
> * HZ: I don't think it is decided how we will run the session. I imagine a lot of conversation will be aound future of CG and WG. Last year there were a lot of converstations. If we have videos they will be mentioned during the session. This way people will have chance to ask questions. In the end I don't know and this is not just up to me. I'm not sure if I will be there in person.

Reviewer: ```suggestion
* RG: What is the plan of CG or Solid team? What will be the structure of this session? I can update work on notifications. If the plan is to talk about administration, it probably will not fit.
* HZ: I don't think it is decided how we will run the session. I imagine a lot of conversation will be around future of CG and WG. Last year, there were a lot of conversations. If we have videos, they will be mentioned during the session. This way people will have a chance to ask questions. In the end, I don't know and this is not just up to me. I'm not sure if I will be there in person.
```

* RG: Can you have a chat with Solid team?
* HZ: Next meeting will be the 2nd week of August.
* RG: I don't know if you have a chair for this session appointed.

ACTION: HZ to find out who will chair the TPAC meeting

### WG status

* HZ: AC vote is not scheduled yet, even though TAG has approved the charter. Vote will probably start in September. 

> * GE: Do you have any more news on WG, did someone speak with PAC?
> * HZ: I didn't, I know that Aaron and Eric have regular meetings. From what I know PAC expected TAG review to end in June and that happened.
> * HZ: Tim should know since he should receive notifications. I assume that people might be on vacations/holidays. 

Reviewer: ```suggestion
* GE: Do you have any more news on WG? Did someone speak with PAC?
* HZ: I didn't. I know that Aaron and Eric have regular meetings. From what I know, PAC expected TAG review to end in June, and that happened.
* HZ: Tim should know, since he should receive notifications. I assume that people might be on vacations/holidays. 
```
